.circleci/config.yml

```diff
@@ -87,6 +87,9 @@ jobs:
     executor: golang-latest
     steps:
       - checkout
+      - run:
+          name: Install syft
+          command: curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin
```

Comment:
Please add this step to the `publish-release` job as well, otherwise I don't think SBOM generation will work for an actual release.